README.md

```diff
@@ -50,6 +50,17 @@ options:
   --download-image  Download images to local
 ```
 
+## Tips
+
+Some wrong will ouccured if you have ever imported a unsanitizer markdwon file such as uploading failly.The console will prompt non-standard documents in red font, and you need to view them manually.
```

Comment:
suggest:

If you have imported illegal Markdwon files (e.g. failed uploads), some error messages will appear. The console will indicate illegal files in red font and you will need to view them manually.